src/components/Card.js

```diff
@@ -42,6 +66,9 @@ export default class Card {
     this._cardPicture.alt = `Photo of ${this._cardTitle}`;
 
     this._likeButton = this._element.querySelector(".card__like-button");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**